mmcls/models/heads/linear_head_gcvit.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyrigforward_trainht (c) OpenMMLab. All rights reserved.
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+
+from mmcls.registry import MODELS
+from .cls_head import ClsHead
+
+
+@MODELS.register_module()
+class LinearClsHead_GCViT(ClsHead):
```

Comment:
This new head seems similar with `LinearClsHead`, maybe you can use the `LinearClsHead` directly.